qdisc_linux.go

```diff
@@ -286,6 +286,12 @@ func qdiscPayload(req *nl.NetlinkRequest, qdisc Qdisc) error {
 		if qdisc.FlowDefaultRate > 0 {
 			options.AddRtAttr(nl.TCA_FQ_FLOW_DEFAULT_RATE, nl.Uint32Attr((uint32(qdisc.FlowDefaultRate))))
 		}
+		if qdisc.Horizon > 0 {
+			options.AddRtAttr(nl.TCA_FQ_HORIZON, nl.Uint32Attr((uint32(qdisc.Horizon))))
+		}
+		if qdisc.HorizonDropPolicy != HORIZON_DROP_POLICY_DEFAULT {
+			options.AddRtAttr(nl.TCA_FQ_HORIZON_DROP, nl.Uint8Attr((uint8(qdisc.HorizonDropPolicy))))
```

Comment:
> HorizonDropPolicy uint8

Why is the `uint8` casting needed?

Reply:
Yeah, looks like it is not needed, neither the `u32` casts above for the `Horizon` (for sure, I just copy-pasted code which was used for other attributes already). I will remove it.